lib/guest_installation_and_configuration_base.pm

```diff
@@ -1254,8 +1254,8 @@ sub config_guest_installation_automation {
         $self->{guest_installation_automation_options} = "--extra-args \"inst.ks=$self->{guest_installation_automation_file}\"";
         $self->{guest_installation_automation_options} = "--extra-args \"ks=$self->{guest_installation_automation_file}\"" if (($self->{guest_os_name} =~ /oraclelinux/im) and ($self->{guest_version_major} lt 7));
     }
-    if (script_run("curl -sSf $self->{guest_installation_automation_file} > /dev/null") ne 0) {
-        record_info("Guest $self->{guest_name} unattended installation file hosted on local host can not be reached", "Mark guest installation as FAILED. The unattended installation file url is $self->{guest_installation_automation_file}");
+    if (script_retry("curl -sSf $self->{guest_installation_automation_file} > /dev/null") ne 0) {
+        record_soft_failure("Guest $self->{guest_name} unattended installation file hosted on local host can not be reached", "Mark guest installation as FAILED. The unattended installation file url is $self->{guest_installation_automation_file}");
```

Comment:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/Makefile#L154 should have failed this. Only use `record_soft_failure` with ticket references

Reply:
Ok, no problem. I'll make some changes.

Reply:
done.

Reply:
@okurz Could you review and approve?